README.md

```diff
@@ -35,6 +35,12 @@ The priority levels for processes, according to the MSDN documentation, are as f
 
 The values on the left in the above table will be the possible values for use as ```process_priority``` in the ```process_config.json``` configuration file.
 
+## Build from source
+1. Install Rust for windows.
+2. clone and unzip. 
+3. cd to the project directory (the one that contains `Cargo.tml`)
+4. run `cargo build`
```

Comment:
I would personally prefer it to be more explicit, i.e.

2. Clone this repository and then extract the zip. (Alternatively: `git clone https://github.com/YemowtRonoc/process-priority-manager`


Also, I would just capitalise the 'r' for "Run" on step 4

Reply:
Does this way looking good?

```suggestion
2. Clone this repository and then extract the zip. (Alternatively: git clone https://github.com/YemowtRonoc/process-priority-manager
3. `cd` to the project directory (the one that contains `Cargo.tml`)
4. Run `cargo build`
```

Reply:
That's exactly what I meant. I'm happy enough to merge that once it's committed